concept.tex

```diff
@@ -597,6 +597,16 @@ \subsubsection{Research Data Management and management of other outputs}
 communication strategies will also be GDPR compliant.
 We have no plans to collect or produce PII during the project.
 
+\noindent \textbf{Software} \\
+
+Most research data form \TheProject will take the form of code.
+All code produced by \TheProject will be developed publicly,
+licensed under OSI-approved Open Source licenses,
+and hosted on public repositories, such as GitHub.
+Milestone achievements of the software may be archived
+in separate repositories such as Zenodo,
+e.g. to coincide with publications.
```

Comment:
There are some repetitions with "1.2.12 Open science practices and implementation". Should "Open science practices and implementation" section be moved here?

Reply:
I don't know. They seem to ask for huge amounts of redundancy. But the DMP part is more about storage/preservation (i.e. hosted on gihtub), whereas 1.2.12 is more about practices/licenses (open source, developed in public).

Reply:
Could try to be more brief here and just mention public hosting and a reference to the open practices.

Reply:
You're right. Maybe make a reference to the previous section? 

You also have a typoe: Most research data form \TheProject   --> Most research data from \TheProject 

Such as:

Most research data from \TheProject will take the form of code and will be developed publicly (see section 1.2.12).
Milestone achievements of the software may be archived
in separate repositories such as Zenodo,
e.g. to coincide with publications.